CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to become a contributor and submit your own code
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
```

Comment:
"the desired repo" = ocean?

Reply:
I was thinking this is more general than just ocean so I have tried to avoid mentioning it explicitly.

Reply:
Well, this document is in ocean, so it should contain instructions for contributing to ocean. Another approach would be to have a general doc -- stored in neptune -- that applies to all our public repos. The doc in ocean could then just link to this.

Reply:
I would think a general doc is a better way to go. This was just an initial prototype in ocean. I can move the PR to neptune though.

Reply:
Sounds good. It'd probably make sense to combine this with https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst.